src/pages/Game/Game.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useEffect } from "react";
+import { Navigate } from "react-router";
+import { useSelector } from "react-redux";
```

Comment:
3 and 4 line u can combine
import { useSelector, useDispatch } from "react-redux";